src/components/DeleteOrg/DeleteOrg.test.tsx

```diff
@@ -114,19 +191,26 @@ describe('Delete Organization Component', () => {
         </BrowserRouter>
       </MockedProvider>
     );
-    fireEvent.click(screen.getByTestId('openDeleteModalBtn'));
-    expect(screen.getByTestId('orgDeleteModal')).toBeInTheDocument();
-    fireEvent.click(screen.getByTestId('closeDelOrgModalBtn'));
-    await waitFor(() => {
-      expect(screen.queryByTestId('orgDeleteModal')).toBeNull();
+    await act(async () => {
+      await new Promise((resolve) => setTimeout(resolve, 1000));
     });
-    expect(window.location).toBeAt('/orgsetting/id=123');
+    screen.getByTestId(/openDeleteModalBtn/i).click();
+    screen.getByTestId(/deleteOrganizationBtn/i).click();
+    await act(async () => {
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Instead of adding the same lines of code everywhere in the tests, you can create an async wait() function in this file, which then you can use over here